src/Microsoft.MonoDevelop.LanguageServices.Razor/Microsoft.MonoDevelop.LanguageServices.Razor.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net46</TargetFrameworks>
+    <Description>Razor is a markup syntax for adding server-side logic to web pages. This package contains the Razor design-time infrastructure for Mono Develop.</Description>
+    <EnableApiCheck>false</EnableApiCheck>
```

Comment:
empty baseline instead?

Reply:
Gonna leave for now. This is how the rest of the VS specific assemblies are interacting with API check today. We'll update them all together when its time 😄

Reply:
Please log an issue for this. It's well past time.

Reply:
We're still making API breaking changes to these assemblies. Are you just suggesting empty baselines for all of the VS specific assemblies? Or to have real baselines?